### PR TITLE
fix HomeBottomAppBar not displaying all button item depending on avai…

### PR DESCRIPTION
**Pull request by FannyDemey**
fix HomeBottomAppBar not displaying all button item depending on available space. improving accessibility at the same time

### CODE REVIEW

File: app/src/main/java/org/scottishtecharmy/soundscape/screens/home/home/HomeBottomAppBar.kt [hunk 1]
```diff
@@ -41,119 +48,127 @@ fun HomeBottomAppBar(
         Toast.makeText(context, notAvailableText, Toast.LENGTH_SHORT).show()
     }
 
-    BottomAppBar(
```
Comment (FannyDemey) [inline app/src/main/java/org/scottishtecharmy/soundscape/screens/home/home/HomeBottomAppBar.kt:44]: getting rid of `BottomAppBar` as this is not really a bottom bar element (just normal button, with no selected state)

File: app/src/main/java/org/scottishtecharmy/soundscape/screens/home/home/HomeBottomAppBar.kt [hunk 2]
```diff
@@ -41,119 +48,127 @@ fun HomeBottomAppBar(
...
+            verticalArrangement = Arrangement.spacedBy(8.dp),
+            modifier = Modifier.padding(bottom = 8.dp)
+        ) {
+            Text(
+                textAlign = TextAlign.Start,
+                text = stringResource(R.string.callouts_panel_title).uppercase(),
+                modifier = Modifier
+                    .padding(start = 16.dp, end = 8.dp)
+                    .semantics {
+                        heading()
```
Comment (FannyDemey) [inline app/src/main/java/org/scottishtecharmy/soundscape/screens/home/home/HomeBottomAppBar.kt:67]: good thing to set heading on things that looks like title, as screen reader user can easily navigate from title to title, skipping everything in between.

File: app/src/main/java/org/scottishtecharmy/soundscape/screens/home/home/HomeBottomAppBar.kt [hunk 3]
```diff
@@ -41,119 +48,127 @@ fun HomeBottomAppBar(
...
+                style = MaterialTheme.typography.labelSmall,
+            )
             Row(
                 verticalAlignment = Alignment.Top,
                 horizontalArrangement = Arrangement.SpaceBetween,
                 modifier =
                 Modifier
-                    .fillMaxWidth(),
+                    .fillMaxWidth()
+                    .height(IntrinsicSize.Max),
```
Comment (FannyDemey) [inline app/src/main/java/org/scottishtecharmy/soundscape/screens/home/home/HomeBottomAppBar.kt:78]: ` .height(IntrinsicSize.Max)` combine with `.fillMaxHeight()` on child, allow us to say all button height should be the height of the button with the biggest height. Then, we can align element more easily.

File: app/src/main/java/org/scottishtecharmy/soundscape/screens/home/home/HomeBottomAppBar.kt [hunk 4]
```diff
@@ -41,119 +48,127 @@ fun HomeBottomAppBar(
...
+        contentPadding = PaddingValues(0.dp)
+    ) {
+        Column(
+            verticalArrangement = Arrangement.Top,
+            horizontalAlignment = Alignment.CenterHorizontally,
+            modifier = Modifier.fillMaxHeight().fillMaxWidth()
+        ) {
+            Icon(
+                painter = icon,
+                contentDescription = null,
```
Comment (FannyDemey) [inline app/src/main/java/org/scottishtecharmy/soundscape/screens/home/home/HomeBottomAppBar.kt:134]: no need of contentDescription as we have a text behind

**Review by davecraig — APPROVED** [review]

**Pull request merged**